app/assets/javascripts/components/articles/article_list.jsx

```diff
@@ -70,7 +63,7 @@ const ArticleList = ({
       table_key="articles"
       className="table--expandable table--hoverable"
       none_message={CourseUtils.i18n('articles_none', course.string_prefix)}
-      store={ArticleStore}
```

Comment:
Is ArticleStore used anywhere in the app still? Since the fetch has moved to redux, we need to make sure ArticleStore is being replaced with redux state everywhere it is used, and the store should be deleted.

Reply:
ArticleStore is also being used substantively by AssignmentList (and required but not needed for anything as far as I can tell in AvailableArticlesList). AssignmentList should be updated to use the `articles` props that it receives, instead of calling ArticleStore to find the article (if any) corresponding to the assignment title, and the usage of ArticleStore can probably just be removed from AvailableArticlesList with no further changes.

Reply:
Can I remove Editable from AssignmentList too?

Reply:
No, I don't think so. It needs it for refreshing the assignment list when AssignmentStore changes.

Reply:
Once AssignmentStore gets converted to redux, then we can remove Editable.

Reply:
yeah, I thought so too.